File: src/Commands/others/ping.ts
import { Command } from "../../Interfaces";

export const command: Command = {
  name: "ping",
  description: "Check if bot is online.",
  aliases: [],
  run: async (client, message, args) => {
    message.reply("Pong! :stuck_out_tongue_winking_eye:");
  }
}

File: src/bot/commands/others/ping.rs
use crate::bot::commands::{Context, Error};

/// Check if bot is online
#[poise::command(slash_command, prefix_command, category = "Others")]
pub async fn ping(ctx: Context<'_>) -> Result<(), Error> {
    message!(normal, ctx, ("Online"); ("Pong! :stuck_out_tongue_winking_eye:"); true);

    Ok(())
}
